Palo Alto Networks/paloalto-cortex/ingest/parser.yml

```
@@ -0,0 +1,37 @@
+name: cortex-xdr
+pipeline:
+  - name: parsed_event
+    description: ''
+    filter: ''
+    external:
+      name: cef.parse-cef
+      properties:
+        input_field: '{{original.message}}'
+        output_field: message
+  - name: setter
+    description: ''
+    filter: ''
+    external: null
+stages:
+  setter:
+    actions:
+      - set:
+          cortexxdr.shost: '{{parsed_event.message.shost}}'
+          cortexxdr.suser: '{{parsed_event.message.suser}}'
```

According to your tests, `suser` contains the username and the user's domain (e.g: `domain\\user`).
I suggest extracting this two information:
```suggestion
          user.name: '{{parsed_event.message.suser.split("\\"}[1]}'
           user.domain: '{{parsed_event.message.suser.split("\\"}[0]}'
```